### PR TITLE
Added method with appropriate ViewModel passing to ProductsInSubType …

### PR DESCRIPTION
**Pull request by Jgregnier**
## Pull Request Description

This method will be handling our new ProductsInSubType View. 
This method will get all the products of a specific subtype and return them to the new view.
## Steps to Test
1. Check for errors.
2. Make sure theres no spelling mistakes.
## Link to Feature Ticket
#111

…view

### CODE REVIEW

**Review by JamsDuke — COMMENTED** [review]
looks good to merge

File: Controllers/ProductsController.cs [hunk 1]
```diff
@@ -68,6 +68,23 @@ public ProductsController (BangazonWebContext ctx)
...
         }
 
+        //Method Name: ProductsInSubType
+        //Purpose of the Method: This method gets all the products in a selected subtype and returns them.
+        //Arguments in Method: Gets the subtypeId from the route.
+        public async Task<IActionResult> ProductsInSubType([FromRoute] int id) 
+        {
+            AllProductsViewModel model = new AllProductsViewModel(context);
+
+            model.Products =  await context.Product.Where(p => p.SubTypeId == id).ToListAsync();
```
Comment (delaineackerman) [inline Controllers/ProductsController.cs]: It's SubProductTypeId

**Comment by HubDub** [comment]
Looks good

**Review by csmalley89 — COMMENTED** [review]
👍

**Review by thrashMoneyVillainare — APPROVED** [review]
Good2Merge

**Pull request merged**